jdk-1.7-parent/annotation/pom.xml

```diff
@@ -42,6 +37,11 @@
 			<groupId>log4j</groupId>
 			<artifactId>log4j</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.reflections</groupId>
+			<artifactId>reflections</artifactId>
+			<version>0.9.10</version>
```

Comment:
Please extract `<reflections.version>` in the project parent pom so it is easier to keep it up-to-date.